### ISSUE TITLE
Idea: implement @headlessui/tailwindcss

### ISSUE THREAD

**Issue by inwardmovement** [rgossiaux/svelte-headlessui#97]
Idea: implement [@headlessui/tailwindcss](https://github.com/tailwindlabs/headlessui/tree/main/packages/%40headlessui-tailwindcss).
It allows to use Tailwind CSS utilities for styling the components based on their state.

Or maybe it should be a separate project?

**Comment by rgossiaux** [comment]
Yeah, I saw that they released this a few days ago. Looks pretty handy! The changes to support the public are relatively small too.

There are a number of other pieces of functionality that I'll be adding first before getting to this, but feel free to follow this issue to get updates for when it gets added to this library.
